src/Core/src/Core/IAbsoluteLayout.cs

```diff
@@ -8,8 +8,18 @@ namespace Microsoft.Maui
 	/// </summary>
 	public interface IAbsoluteLayout : ILayout
 	{
+		/// <summary>
+		/// Gets the layout bounds of a View.
+		/// </summary>
+		/// <param name="view">A visual element.</param>
+		/// <returns>The layout bounds of the object.</returns>
 		Rect GetLayoutBounds(IView view);
 
+		/// <summary>
+		/// Gets the layout flags that were specified when view was added to an AbsoluteLayout.
```

Comment:
The flags can change even after the IView has been added. Maybe "Gets the layout flags of the IView in the IAbsoluteLayout."?

Reply:
I just updated comments. Thanks for the feedback.